config/routes.rb

```diff
@@ -18,9 +18,11 @@
     resources :user_sessions, only: :destroy
     resources :users, only: :destroy
     post "logout" => "user_sessions#destroy", :as => :logout
-
+    
+    get '/flickr_search/search_photos', to: 'flickr_search#search_photos', as: :search_photos
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.